### PR TITLE
Adrevenue - new article replacing the adrevenue connector

### PR DESCRIPTION
**Pull request by dori-af**
@liaz-af

### CODE REVIEW

File: docs/ad-revenue-unity.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+```c#
+Dictionary<string, string> additionalParams = new Dictionary<string, string>();
+additionalParams.Add("value1", "5");
+additionalParams.Add(AdRevenueScheme.COUNTRY, "USA");
+additionalParams.Add(AFAdRevenueEvent.AD_UNIT, "89b8c0159a50ebd1");
+additionalParams.Add(AFAdRevenueEvent.AD_TYPE, "Banner");
+additionalParams.Add(AFAdRevenueEvent.PLACEMENT, "place");
+var logRevenue = new AFAdRevenueData("monetizationNetworkEx", MediationNetwork.GoogleAdMob, "USD", 0.99);
+AppsFlyer.logAdRevenue(logRevenue, parameters);
```
Comment (liaz-af) [inline docs/ad-revenue-unity.md:38]: AppsFlyer.logAdRevenue(logRevenue, additionalParams);

File: docs/ad-revenue-unity.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+Send ad revenue data from the impression payload to AppsFlyer regardless of the mediation network you use.
+
+| parameter            | type                                   | description                      |
+| -------------------- | -------------------------------------- | -------------------------------- |
+| monetizationNetwork  | string                                 | monetization network             |
+| mediationNetwork     | AppsFlyerAdRevenueMediationNetworkType | Enum for mediaton network type   |
+| eventRevenue         | string                                 | event revenue                    |
+| revenueCurrency      | string                                 | revenue currency                 |
+| additionalParameters | Dictionary\<string, string>            | Any custom additional parameters |
```
Comment (liaz-af) [inline docs/ad-revenue-unity.md:133]: It is `Dictionary<string, string>`

Comment (dori-af) [reply docs/ad-revenue-unity.md:133]: fixed

File: docs/ad-revenue-unity.md (hunk 1 above)
Comment (dori-af) [reply docs/ad-revenue-unity.md:38]: fixed

**Pull request merged**